src/Core/src/Core/IView.cs

```diff
@@ -109,11 +109,6 @@ public interface IView : IElement, ITransform
 		/// </summary>
 		new IViewHandler? Handler { get; set; }
 
-		/// <summary>
-		/// Gets the Parent of the Element.
-		/// </summary>
-		new IView? Parent { get; }
-
```

Comment:
This property does not seem to be useful. Views could have a window as a parent so we would get a false null here. It also does not seem to be used.